docs/OperationGuide/index.rst

```diff
@@ -0,0 +1,42 @@
+Operation Guide
+=================
+
+This document is the fourth installment in a series of guides detailing
+how to set up and operate the Reporting Tool. Its role is to lay out a
+step-by-step guide on how to onboard the FLINT’s Flux Output Database
+into the Reporting Tool and have it processed and aggregated to give you
+Emissions Graphs and Reports. It assumes that you have already installed
+the Reporting Tool as detailed in the third guide.
+
+Prerequisites
+-------------
+
+We need to meet two preconditions in order to successfully operate the
+Reporting Tool:
+
+== =====================================================
+\  Prerequirement
+== =====================================================
+1  Ascertaining that we have the right to log in to the server and perform the operational actions 
+2  Ascertaining that we have a compatible Flux Output Database Format to work with 
+3  A shell Terminal to work from (the default in Linux or Mac and Powershell in Windows)
+== =====================================================
+
+Outline
+-------
+
+There are four key steps to successfully operating the Reporting Tool:
+
+== ==============================================================
+\  Prerequirement
+== ==============================================================
+1  Check that all Reporting Tool Services is Up and Running
+2  Upload a Flux Output Database Backup to the Reporting Tool
+   Application Server
+3  Onboard the Flux Output Database Backup
+4  Monitoring the Database Processing Event
+== ==============================================================
+
+We will explore each of these steps in detail in the sections that
+
+follow
```

Comment:
This documentation piece seems incomplete. Is that intentional or is this PR WIP?

Reply:
**ya like in the doc file of operation guide it was concluded like this & I had no idea of what should I put next as there was a lot of stuff which was to be added after this** 

 and also I did not initiate the make clean && make HTML command as I was not sure about this so once you approve I would make another PR containing the final commit